libs/request.js

```diff
@@ -54,38 +54,31 @@ module.exports = function (options = {}) {
 
   if (options.debug) requestdebug(request)
 
-  const requestOptions = {}
-
-  requestOptions.json = options.json
-  requestOptions.jar = options.jar
-  requestOptions.headers = options.headers
-  requestOptions.followAllRedirects = options.followAllRedirects
-  requestOptions.strictSSL = options.strictSSL
-
   if (options.cheerio) {
-    requestOptions.transform = function (body, response, resolveWithFullResponse) {
-      let result = require('cheerio').load(body)
+    options.transform = function (body, response) {
+      // memory issues https://github.com/cheeriojs/cheerio/issues/830
+      let result = require('cheerio').load(response.body)
```

Comment:
If this solves the memory issue, that is cool.

Reply:
Not sure, actually. :/